Description: Use python-six instead of embedded copy
Author: Sebastian Ramacher <sramacher@debian.org>
Last-Update: 2013-12-09

--- a/ecdsa/der.py
+++ b/ecdsa/der.py
@@ -2,7 +2,7 @@
 
 import binascii
 import base64
-from .six import int2byte, b, integer_types, text_type
+from six import int2byte, b, integer_types, text_type
 
 class UnexpectedDER(Exception):
     pass
--- a/ecdsa/ecdsa.py
+++ b/ecdsa/ecdsa.py
@@ -53,7 +53,7 @@
 Written in 2005 by Peter Pearson and placed in the public domain.
 """
 
-from .six import int2byte, b, print_
+from six import int2byte, b, print_
 from . import ellipticcurve
 from . import numbertheory
 import random
--- a/ecdsa/ellipticcurve.py
+++ b/ecdsa/ellipticcurve.py
@@ -34,7 +34,7 @@
 
 from __future__ import division
 
-from .six import print_
+from six import print_
 from . import numbertheory
 
 class CurveFp( object ):
--- a/ecdsa/keys.py
+++ b/ecdsa/keys.py
@@ -7,7 +7,7 @@
 from .util import string_to_number, number_to_string, randrange
 from .util import sigencode_string, sigdecode_string
 from .util import oid_ecPublicKey, encoded_oid_ecPublicKey
-from .six import PY3, b
+from six import PY3, b
 from hashlib import sha1
 
 class BadSignatureError(Exception):
--- a/ecdsa/numbertheory.py
+++ b/ecdsa/numbertheory.py
@@ -11,8 +11,8 @@
 
 from __future__ import division
 
-from .six import print_, integer_types
-from .six.moves import reduce
+from six import print_, integer_types
+from six.moves import reduce
 
 import math
 
--- a/ecdsa/rfc6979.py
+++ b/ecdsa/rfc6979.py
@@ -12,7 +12,7 @@
 import hmac
 from binascii import hexlify
 from .util import number_to_string, number_to_string_crop
-from .six import b
+from six import b
 
 try:
     bin(0)
--- a/ecdsa/test_pyecdsa.py
+++ b/ecdsa/test_pyecdsa.py
@@ -8,7 +8,7 @@
 from binascii import hexlify, unhexlify
 from hashlib import sha1, sha256, sha512
 
-from .six import b, print_, binary_type
+from six import b, print_, binary_type
 from .keys import SigningKey, VerifyingKey
 from .keys import BadSignatureError
 from . import util
--- a/ecdsa/util.py
+++ b/ecdsa/util.py
@@ -6,7 +6,7 @@
 from hashlib import sha256
 from . import der
 from .curves import orderlen
-from .six import PY3, int2byte, b, next
+from six import PY3, int2byte, b, next
 
 # RFC5480:
 #   The "unrestricted" algorithm identifier is:
